sources/tools/Stride.VisualStudio.Package/MyControl.xaml.cs

```diff
@@ -23,7 +23,7 @@ public partial class MyControl : UserControl
     {
         public MyControl()
         {
-            InitializeComponent();
+            //InitializeComponent();
```

Comment:
Isn't this `MyControl` class a simple example control from the template? It does nothing but to show a button that says "Click me!".

I think it should be removed.

Reply:
removed this as well as MyToolWindow